server/controllers/userController.js

```diff
@@ -20,13 +23,58 @@ export default class UserController {
   */
   static create(req, res) {
     const { username, email, password } = req.body;
+
+    // generate verification token and store in verifyToken variable
+    const verifyToken = crypto.randomBytes(16).toString('hex');
+
     bcrypt.hash(password, saltRounds)
-      .then(hashedPassword => User.create({ username, email, password: hashedPassword, })
```

Comment:
I could put everything on one line. eslint flag it to be more than 100 characters in one line